RapidReact/src/main/java/frc/robot/subsystem/ClimberSubsystem.java

```diff
@@ -329,6 +330,34 @@ public void periodic() {
     climberRightPosition.log(LogLevel.GENERAL, climberRight.getSelectedSensorPosition());
     climberRightError.log(LogLevel.GENERAL, climberRight.getClosedLoopError());
 
+    boolean climbLeftRevLimitSwitchClosed = climberLeft.getSensorCollection().isRevLimitSwitchClosed();
+    boolean climbRightRevLimitSwitchClosed = climberRight.getSensorCollection().isRevLimitSwitchClosed();
+
+    if (!climbLeftRevLimitSwitchClosed)
+    {
+      if (!climbLeftEncoderZeroed)
+      {
+        climberLeft.set(ControlMode.PercentOutput, -0.1);
+      }
+    }
+    else
+    {
+      climbLeftEncoderZeroed = true;
```

Comment:
As far as I can tell, this just sets climbLeftEncoderZeroed, and doesn't actually zero the encoder. Does the encoder get zeroed somewhere else?

Also, don't you need to stop the motor when you're done zeroing?